cmake_modules/FindLz4.cmake

```diff
@@ -25,7 +25,7 @@ find_path(LZ4_INCLUDE_DIR lz4.h
   # make sure we don't accidentally pick up a different version
   NO_CMAKE_SYSTEM_PATH
   NO_SYSTEM_ENVIRONMENT_PATH)
-find_library(LZ4_STATIC_LIB liblz4.a
+find_library(LZ4_STATIC_LIB NAMES liblz4_pic.a liblz4.a
```

Comment:
What does the _pic library do?

Reply:
pic library has position independent code generation. We have 2 libraries _pic.a and .a for each tp2 library. When we compile debug mode kuduraft, it complains of unhandle-able relocations, which essentially means that you need to link with PIC code.